src/sonic-yang-models/yang-models/sonic-sflow.yang

```diff
@@ -43,17 +47,34 @@ module sonic-sflow{
                     type string {
                         length 1..64;
                     }
+                    description "Name of the Sflow collector";
                 }
 
                 leaf collector_ip {
                     mandatory true;
                     type inet:ip-address;
+                    description "IPv4/IPv6 address of the Sflow collector";
                 }
 
 
                 leaf collector_port {
                     type inet:port-number;
                     default 6343;
+                    description "Destination L4 port of the Sflow collector";
+                }
+
+                leaf collector_vrf {
+                    must "(current() != 'mgmt') or (/mvrf:sonic-mgmt_vrf/mvrf:MGMT_VRF_CONFIG/mvrf:vrf_global/mvrf:mgmtVrfEnabled = 'true')" {
```

Comment:
This condition allows only mgmt VRF? but below pattern allows 'default' VRF as well.

Reply:
This condition will check if mgmt vrf is enabled when the current collector_vrf is configured as 'mgmt'. If not it will error.
For default vrf there is no such requirement.

Reply:
OK

Reply:
Hope the sflow backend is already handling the case with 'default' VRF config, please check and update.

Reply:
Yes it does https://github.com/sflow/host-sflow/blob/d78a5f5dce73fb77c093795be7a96cf12220c3c6/src/Linux/mod_sonic.c#L1422